packages/fx-core/src/common/deps-checker/depsManager.ts

```diff
@@ -47,14 +47,14 @@ export class DepsManager {
    * @param options If fastFail is false, it will continue even if one of the dependencies fails to install. Default value is true.
    */
   public async ensureDependencies(
```

Comment:
Can we add a new `ensureDependency(dependency: DepsType, options: {version: number})` for v3 and do not change the old `ensureDependencies`

Reply:
Or can we just add the vx version in `DepsOptions`?

Reply:
Added `ensureDependency()`. I think if just add a vx version in `DepsOptions`, it would be hard to pass to the generic `DepsChecker.resolve(options)`